src/config/scrivitoTenants.ts

```diff
@@ -0,0 +1,36 @@
+export function isMultitenancyEnabled(): boolean {
+  return !import.meta.env.SCRIVITO_TENANT
+}
+
+const location = typeof window !== 'undefined' ? window.location : undefined
+
+export function scrivitoTenantId(): string | undefined {
```

Comment:
I think it would be better, if we switch to 
```suggestion
export function scrivitoTenantId(): string {
```

and throw, if no tenantId could be calculated. This would help to avoid "if (!tenant)" code ouside of `scrivitoTenantId()`.

Reply:
See also discussion in https://github.com/Scrivito/scrivito-portal-app/pull/354#discussion_r1560918091.